neuromation/api/client.py

```diff
@@ -103,3 +106,27 @@ def _get_session_cookie(self) -> Optional["Morsel[str]"]:
             if cookie.key == "NEURO_SESSION":
                 return cookie
         return None
+
+    async def _get_quota_info(self, user: Optional[str] = None) -> _QuotaInfo:
```

Comment:
Please keep `client._quota` private API.
There is no need to mess `client.py` with quota-specific things.

Reply:
OK